pyproject.toml

```diff
@@ -41,7 +43,8 @@ pytz = "^2022.6"
 tzlocal = "^4.2"
 mxcubecore = ">=1.102.0"
 mxcube-video-streamer = ">=1.0.0"
-bcrypt = "^4.0.1"
+bcrypt = "4.0.1"
+authlib = "^1.3.0"
```

Comment:
As far as I can tell, we do not directly depend on those (`bcrypt` and `authlib`). Can we have a comment explaining why we add those as direct dependencies?

Reply:
Its for the SSO-keycloak integration. I tested everything with that development and I did not remove the dependencies specific to SSO. Ill remove these for the being.

Reply:
Seems fixed